src/index.html

```diff
@@ -10,6 +10,7 @@
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/Swiper/4.4.2/css/swiper.min.css">
   <link rel="manifest" href="manifest.json">
   <meta name="theme-color" content="#0d0d63">
+  <script type="text/javascript" src="https://smtpjs.com/v3/smtp.js"></script>
```

Comment:
Why using external API?

Reply:
@simsausaurabh  As we know we don't have backend so, I am using smpt.js API which is free to send contact form response. Earlier form was not sending responses .

Reply:
@simsausaurabh  Secure token is used , I cannot my raw gmail login.

Reply:
But we are not supposed to display these tokens openly.

Reply:
Please also take review from co-developers on Gitter